schema.go

```diff
@@ -364,31 +381,32 @@ func (conn *Connection) loadSchema() (err error) {
 		}
 	}
 
+	checkNamesSupported(conn, schema)
+
 	conn.lockShards()
 	conn.Schema = schema
+	conn.schemaResolver = schema
 	conn.unlockShards()
 
 	return nil
 }
 
-// ResolveSpaceIndex tries to resolve space and index numbers.
+// ResolveSpace tries to resolve a space number.
 // Note: s can be a number, string, or an object of Space type.
-// Note: i can be a number, string, or an object of Index type.
-func (schema *Schema) ResolveSpaceIndex(s interface{}, i interface{}) (uint32, uint32, error) {
+func (schema *Schema) ResolveSpace(s interface{}) (uint32, error) {
 	var (
-		spaceNo, indexNo uint32
-		space            *Space
-		index            *Index
-		ok               bool
+		spaceNo uint32
+		space   *Space
+		ok      bool
 	)
 
 	switch s := s.(type) {
 	case string:
 		if schema == nil {
-			return spaceNo, indexNo, fmt.Errorf("Schema is not loaded")
+			return spaceNo, fmt.Errorf("schema is not loaded")
```

Comment:
No, its not. Right now we use default schemaResolver implementation, that will not call Resolver in case of a string `space` or `index` (before actually loading the schema).